Aptoide/aptoide-client-v8
# Made the requested changes on the old pull-request; This involved del…

**What does this PR do?**

This PR aims to implement the last changes to the original appview pull request;

**Database changed?**

 No

**Where should the reviewer start?**

- [ ] ScreenshotsAdapter.java

**How should this be manually tested?**

Some displayable/widget pairs were remove. Check the app to make sure their removal has no side effects;

**Code Review Checklist**
